src/cameras/CubeCamera.js

```diff
@@ -3,6 +3,7 @@
  *	- renders scene into axis-aligned cube
  *
  * @author alteredq / http://alteredqualia.com/
+ * @author prafullit
```

Comment:
Add your name as an author only if you feel your contribution is significant, and you are willing to be responsible for the file maintenance.